app/views/expenses/CheckEmploymentExpensesView.scala.html

```diff
@@ -39,16 +39,12 @@
 @layout(pageTitle = messages(s"checkYourExpenses.heading.${if(user.isAgent) "agent" else "individual"}"), taxYear = Some(taxYear), isAgent = user.isAgent) {
     @heading(
         messages(s"checkYourExpenses.heading.${if(user.isAgent) "agent" else "individual"}"),
-        Some(messages(s"employment.caption", (taxYear - 1).toString, taxYear.toString)),
+        Some(messages(s"expenses.caption", (taxYear - 1).toString, taxYear.toString)),
         "govuk-!-margin-bottom-3"
     )
 
     <p class="govuk-body">
-        @{if(isMultipleEmployments){
-            messages(s"checkYourExpenses.sub-heading.${if(user.isAgent) "agent" else "individual"}.multiple")
-        } else {
-            messages(s"checkYourExpenses.hmrcOnlyInfo.${if(user.isAgent) "agent" else "individual"}")
-        }}
+        @messages(s"checkYourExpenses.sub-heading.${if(user.isAgent) "agent" else "individual"}")
```

Comment:
i think you can remove the "isMultipleEmployments" entirely from this page now. Doesn't look like its used anymore now you've remove it from here.

Reply:
Good spot. Thank you